talentprotocol/web-dapp
# Social graph

## Summary

Submission for Social Graph [bounty](https://talentprotocol.notion.site/Social-Graph-2b177eda510d4869b0b045fb4b5178e5)

## How to test?

Install dependencies
`yarn install`

Social Graph should render below Community section in Talent Profile.

## Notes

Currently fetching only as many connections as `connections` endpoint returns.
